scripts/matlab2python/mat2py_listnumpy.m

```diff
@@ -12,7 +12,11 @@
 
    % Convert MATLAB arrays to NumPy arrays
    for i = 1:numel(matlab_arrays)
-       np_array = py.numpy.array(matlab_arrays{i});  % Convert to column vector and then to NumPy array
+       if isrow(matlab_arrays{i})
```

Comment:
It is indeed to ensure that one dimensional data are saved correctly! py.numpy.array automatically creates an column vector in python when not using double {}. Or is this a bit unnecessary and do you propose a more efficient alternative?

Reply:
I am not sure if I understand you correctly but matlab_arrays doesn't need to be an array but can also be a matrix (in case of IMU data or if I want to store a multi-channel output). My concern is that when my output is e.g. 1 row (let's say a week aggregate or a segment aggregate) with 4 channel arrays py.numpy.array creates a column vector instead of a row vector. This is a unique case but for my output it can happen.

Reply:
I see, in case where `matlab_arrays` might contain more than one element and each is a row maybe a code like this would be easier to read
```
for i = 1:numel(matlab_arrays)
        % Ensure MATLAB array is in 2D form
        matlab_array = matlab_arrays{i};
        if isrow(matlab_array)
            matlab_array = matlab_array.';  % Transpose to make it a column vector
        end
        
        % Convert and append to Python list
        np_array = py.numpy.array(matlab_array);
        py_arrays.append(np_array);
    end
````

Reply:
I also thought about something like this but this transposes it to a column vector and stores it in a column vector. But e.g. when my data is 1 row with 4 columns (channels) this will result in a loading error when using tsdf again because it is saved as a 4x1 array while it actually should contain 4 rows with 4 channels according to the metadata. I agree that my solution is not the most efficient one but it solves this particular problem.

Reply:
Thanks for the clarification, then I recommend merging it as it is.